### ISSUE TITLE
Crash on frame rate switch (Google Android TV)

### ISSUE THREAD

**Issue by gmjh123** [vonH/plugin.video.iplayerwww#379]
Hi. Firstly thanks very much indeed for your add-on, I've been using it for many years and only very recently had any issues. As well as the issue I posted earlier about the fire stick, I'm having another separate issue on a Google Android TV box (Onn 4k pro) I suspect that this issue is something you have no control over, but thought I'd check. 

Basically when I try and play something on the BBC add-on, Kodi 21 tries to change the frame rate to 50hz, can't do it, and crashes completely. I can play the add-on fine if I disable Kodi's ability to switch refresh rates and resolution, but then all of the streams on your add-on are in the wrong resolution and the wrong frame rate. No other stream or add-on has this issue, so I'm wondering if you can help as to why it only does it with the BBC add on. Thanks very much again.

**Comment by CaptainTK** [comment]
First of all, that is a function outside of this add-on. This add-on parses the website, extracts a playable manifest URL and passes this URL to the Kodi Player/Inputstream Adaptive.

The Kodi Player/Inputstream Adaptive will then load the manifest, and only then we will know the framerate of the stream. Thus, there is nothing that the add-on can do about the framerate setting.

Having said that, crashes can be intermittent and caused by all kinds of things. I suggest you first extract the kodi.log file from the device right after a crash like this. Hopefully, there should be some lines at the end which include information like "Fatal signal 6 (SIGABRT), code -6 (SI_TKILL)" and point in the direction of where the crash may be originating.

I will close here nonetheless, because chances the add-on can do anything about it are practically zero.

**Comment by gmjh123** [comment]
Ok thanks very much. I'm presuming that as Kodi completely crashes I have to upload the 'Kodi old' log file. Which is here 
https://pastebin.com/Ap2AgzGk
Looking at the end of the log file,I can't see any reference to your add on and I can't see any errors but maybe you'll be able to spot something. Thanks very much

**Comment by gmjh123** [comment]
I've found what the issue is, and it's nothing to do with your add-on. The problem is that my onn 4K Pro box will not switch to the BBC resolution of 1280 x 720. The resolution 1280 x 720 is listed in the list of compatible resolutions for my 4K Pro box and is listed in the white list on Kodi, but for some reason either this box or Kodi won't switch to it. I found this out when trying to switch the UI of Kodi to 1280 x 720 and it crashed. So this issue can be closed again
